### ISSUE TITLE
Telegram bot /profit doesn't show total profit %

### ISSUE THREAD

**Issue by Theagainmen** [freqtrade/freqtrade#2362]
## My environment

* Operating system: Windows
* Python Version: Python 3.7.4
* Branch: freqtrade develop-7d1f66cc.

### The issue
The `/profit` command on the telegram bot shows your total profit, however it's not showing the total profit percentage but something else (I guess cumulative).
Here is how I know:
- My `dry_run_wallet` is `0.0136722676 BTC` roughly €106 (dry mode).
- It shows now:
```
ROI: Close trades
* 0.00016035 BTC (2.34%)
* 1.242 EUR
```
This 2.34% is weird, because 2.34% of €106 should be around €2,48.
Then looking at the BTC value itself (because the EUR value changes a lot), `0.00016035` of `0.0136722676` is `1.17%`. So I don't know why it shows 2.34%.

**Comment by xmatthias** [comment]
> 0.00016035 BTC

This is the sum of all closed trades where `is_open = False` in the database.

> 2.34%

This is the (rounded) mean of all closed trades. it's not cumulative, but an average value (sum of all percentages / number of closed trades).

Now the `/profit` does include another section for all trades, which are calculated identically, but include open trades.

Changing this to mean was done intentionally #869 - reasons for it and calculations are explained in the corresponding issue: #629.

To summarize: Using sum is wrong as soon as you use more than one stake in parallel, or change the stake-amount without dropping the database.

**Comment by Theagainmen** [comment]
Ah of course that makes sense, it shows the average profit percentage per trade, I still rather see the profit percentage compared to my total stake amount. 
Like in the issue https://github.com/freqtrade/freqtrade/issues/629 you posted:
```Correct calculation should show something as follows:
Total investment: 0.6 BTC
profit: 0.00454026
profit-Percentage: 0.756%
```
that's exactly what I would like to see, it would make more sense with the `/profit` command, maybe the average profit per trade should be shown in another command like `/performance` or somewhere else.

**Comment by Theagainmen** [comment]
Could it be that in backtesting the total profit is also wrong?
My dry run wallet value is: `0.0136722676` with a stake_amount of `0.0013672268` (basically 10%).
I just checked and it seems to be wrong, here a screenshot of 2 strategies results: https://imgur.com/a/wqGe0iV

First one has a tot profit of `107.85 %`, so that should be `107,85 %` of my dry wallet value (`0.0136722676`) is `0.014745540606599`, however it shows `0.00590401 BTC`.

Then `0.00590401 BTC` is `43.18 %` of `0.0136722676` and not `107,85 %` (as shown in the screenshot)? Am I missing something? Does tot profit not use your initial wallet value?

**Comment by xmatthias** [comment]
i think your misunderstanding the "dry_run_wallet" value.

it's not used during backtesting.
it's only used as return-value for `/balance` and some balance-related calculations during dry-run. It's not used once dry_run is switched off, and it's not used during backtesting.

this makes the basis for your calculations off.

Backtesting uses `stake_amount` as basis. So if your stake is 0.01 BTC, then you need to calculate profit against this, not against 0.0136, because this is your "dry_run_wallet" setting.

The same is true for `/profit` reports - which are also calculated against the value you put into a trade, which may or may not be identical to your balance in the account.

Your balance is not relevant during backtesting - i may have 1000 BTC in my account, but only allow the bot to trade 1 BTC split up to 3 trades. Backtesting does not require (nor use) your api keys, simply because it does not need to know your balance.

Hope this helps to clarify things.

**Comment by Theagainmen** [comment]
Ah that makes sense indeed, so only the stake_amount you put into trades will be used in the calculations of the tot profit?
So if I have 2 trades:
* $10 make 20% so +$2
* $100 make 5% so +$5 
* Total trade amount input is $110 and in the end I have $117
My tot profit is `(117-110)/110*100 = 6,36%`? 
Indeed.. your total wallet balance wouldn't make sense, my bad!

**Comment by xmatthias** [comment]
but how can you have 2 trades (one with $10, one with $100 as stake)?

you only have 1 stake-amount - yes edge allows something similar - but it won't be 100 / 10, but more like 13$ / 9$ or so ... ).

However, this is considered.
profit percent is calculated per trade (`calc_profit_percent()` - including fees) - taking the mean of the results of that.

https://github.com/freqtrade/freqtrade/blob/7a2d917c66a655865ada64ab7d26d1115951d2c8/freqtrade/persistence.py#L380-L395

**Comment by hroff-1902** [comment]
I read the above in details, the only point that can be considered here as open is the following one:

> ```
> Total investment: 0.6 BTC
> profit: 0.00454026
> profit-Percentage: 0.756%
> ```
> 
> that's exactly what I would like to see, it would make more sense with the `/profit` command, maybe the average profit per trade should be shown in another command like `/performance` or somewhere else.

@Theagainmen  @xmatthias 
So the question is:
Shall we close it or shall consider later as a proposal for adding the "Total investment:" field in the output of the `/profit` command?

**Comment by xmatthias** [comment]
well "Total investment:" is misleading to me.

assume my bot did 20 trades - i have `max_open_trades=3; stake_amount=10USDT`.

what's my total investment? 30 USDT - or 200 USDT? i never invested more than 30 USDT at the same time.

**Comment by xmatthias** [comment]
I would consider this closed by #2661. 
The `/balance` command now shows the current balance also for dry-runs - so the current state including profit / loss can easily be seen from there.

Feel free to reopen if you feel otherwise.
